src/robust.jl

```diff
@@ -21,10 +21,9 @@ function uplo(x::AbstractVector; prop::Real=0.0, count::Integer=0)
     end
 
     # indices for lowest count values
-    x2 = copy(x)
-    lo = partialsort!(x2, 1:count+1)[end]
-    # indices for largest count values
-    up = partialsort!(x2, n-count:n)[1]
+    x2 = Base.copymutable(x)
+    lo = partialsort!(x2, count+1)
```

Comment:
And what if we just did this?
```suggestion
    lo = partialsort(x2, count+1)
```

Reply:
Sorry that doesn't work.